samza-api/src/main/java/org/apache/samza/system/SystemAdmin.java

```diff
@@ -114,4 +114,13 @@ default boolean createStream(StreamSpec streamSpec) {
   default void validateStream(StreamSpec streamSpec) throws StreamValidationException {
     throw new UnsupportedOperationException();
   }
+
+  /**
+   * Check if the stream described by the spec already exists.
+   * @param streamSpec The spec, or blueprint for the physical stream on the system.
+   * @return true if stream exists already, false otherwise
+   */
+  default boolean existStream(StreamSpec streamSpec) {
```

Comment:
Don't have context on this PR, just wondering why we need a new method in the SystemAdmin API for this? Is this just for convenience or do we not currently have a way to check if a stream exists?